references/cuegen/testdata/invalid/default.go

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Perhaps we can merge the test cases into one file and use the struct name to distinguish.

Reply:
This may can't be merged, because Generator will return directly when it encounters an error. If merged, Generator ends after encountering the first struct.